packages/commonwealth/test/integration/api/devnet/cosmos/proposalTx.spec.ts

```diff
@@ -186,47 +170,134 @@ describe('Proposal Tests', () => {
+      expect(resp.rawLog).to.not.be.undefined;
+      expect(resp.rawLog).to.include('VOTE_OPTION_ABSTAIN');
+      expect(resp.rawLog).to.not.include('VOTE_OPTION_YES');
     });
     it('votes YES on an active proposal', async () => {
       await waitOneBlock();
-      const lcd = await setupLCD();
-      const { signerAddress } = await setupSigner();
 
       const { proposals: activeProposals } = await lcd.cosmos.gov.v1.proposals({
         proposalStatus: ProposalStatus.PROPOSAL_STATUS_VOTING_PERIOD,
         voter: '',
         depositor: '',
       });
 
-      chai.assert.isDefined(activeProposals);
-      chai.assert.isAtLeast(activeProposals.length, 1);
+      expect(activeProposals).to.not.be.undefined;
+      expect(activeProposals.length).to.be.greaterThan(0);
+
       const proposal = activeProposals[0];
 
       const msg = encodeMsgVote(
-        signerAddress,
+        signer,
         proposal.id,
         VoteOption.VOTE_OPTION_YES
       );
       const resp = await sendTx(msg);
 
-      chai.assert.isDefined(resp.transactionHash);
-      chai.assert.isDefined(resp.rawLog);
-      chai.assert.isTrue(resp.rawLog.includes('VOTE_OPTION_YES'));
-      chai.assert.isFalse(resp.rawLog.includes('VOTE_OPTION_NO'));
+      expect(resp.transactionHash).to.not.be.undefined;
+      expect(resp.rawLog).to.not.be.undefined;
+      expect(resp.rawLog).to.include('VOTE_OPTION_YES');
+      expect(resp.rawLog).to.not.include('VOTE_OPTION_NO');
+    });
+  });
+
+  // Cosmos gov v1beta1 query tests
+  describe('Cosmos Governance v1 util Tests', () => {
+    describe('csdk', () => {
+      let lcd: LCD;
+      const id = 'csdk';
+      beforeEach(async () => {
+        lcd = await getLCDClient(`http://localhost:8080/cosmosLCD/${id}`);
+      });
+
+      describe('getActiveProposals', () => {
+        it('should fetch active proposals', async () => {
+          const proposals = await getActiveProposalsV1(lcd);
+          expect(proposals.length).to.be.greaterThan(0);
+
+          proposals.forEach((proposal) => {
+            expect(proposal.state.completed).to.eq(false);
+            expect(proposal.state.status).to.be.oneOf([
+              'VotingPeriod',
+              'DepositPeriod',
+            ]);
+            expect(proposal.state.tally).to.not.be.null;
+          });
+        });
+      });
+      describe('getCompletedProposals', () => {
+        it('should fetch completed proposals', async () => {
+          const proposals = await getCompletedProposalsV1(lcd);
+
+          proposals.forEach((proposal) => {
+            expect(proposal.state.completed).to.eq(true);
+            expect(proposal.state.status).to.be.oneOf([
+              'Passed',
+              'Rejected',
+              'Failed',
+            ]);
+          });
+        });
+      });
     });
   });
 });
+
+// TODO: use a v1beta1 devnet for CI testing. See: https://github.com/hicommonwealth/commonwealth/issues/3563
+
```

Comment:
these work locally, but not in CI. I will convert them to a v1beta1 devnet in the next issue, as noted